tests/adl/test_adl_utils.cairo

```diff
@@ -221,18 +225,66 @@ fn given_non_valid_position_when_create_adl_order_then_fails() {
 
 #[test]
 fn given_normal_conditions_when_create_adl_order_then_works() { // Setup
-    //let (caller_address, role_store, data_store, event_emitter, oracle) = setup_oracle_and_store();
+    let (caller_address, role_store, data_store, event_emitter, oracle) = setup_oracle_and_store();
     // TODO 
     // For testing "position_utils::get_position_key",  ".data_store.get_position" should be implmented
-    assert(true, 'e');
+    let account1 = 'account1'.try_into().unwrap();
+    let market = 'market'.try_into().unwrap();
+    let collateral_token = 'token'.try_into().unwrap();
+    let params = adl_utils::CreateAdlOrderParams {
+        data_store: data_store,
+        event_emitter: event_emitter,
+        account: account1,
+        market: market,
+        collateral_token: collateral_token,
+        is_long: true,
+        size_delta_usd: 0,
+        updated_at_block: 100
+    };
+    adl_utils::create_adl_order(params);
```

Comment:
```suggestion
    let key = adl_utils::create_adl_order(params);

    // Assertions
    let order = data_store.get_order(key);
    assert(order.account == account1, 'wrong order');
    assert(order.order_type == OrderType::MarketDecrease(()), 'wrong type');
    assert(order.updated_at_block == 100, 'wrong updated');
```

Reply:
alright, fixing that now, is that all the changes that's needed.